docs/protocol/src/SUMMARY.md

```diff
@@ -17,7 +17,6 @@
     - [Group Hash](./crypto/decaf377/group_hash.md)
     - [Test Vectors](./crypto/decaf377/test_vectors.md)
   - [Randomizable Signatures](./crypto/decaf377-rdsa.md)
-  - [Key Agreement](./crypto/decaf377-ka.md)
```

Comment:
we should actually keep this section and link to the [implementation](https://github.com/penumbra-zone/decaf377-ka)